crates/oxide/src/extractor/css_variable_machine.rs

```diff
@@ -135,6 +135,13 @@ const CLASS_TABLE: [Class; 256] = {
     set_range!(Class::AllowedCharacter, b'A'..=b'Z');
     set_range!(Class::AllowedCharacter, b'0'..=b'9');
 
+    // non-ASCII (such as Emoji): https://drafts.csswg.org/css-syntax-3/#non-ascii-ident-code-point
+    let mut i = 0x80;
+    while i <= 0xff {
+        table[i as usize] = Class::AllowedCharacter;
+        i += 1;
+    }
```

Comment:
Can this also use the `set_range!` macro? 

```rs
    set_range!(Class:: AllowedCharacter, 0x80..=0xff)
```

Reply:
Ah yep! I was dealing with `\u{00B7}` values prior to this which could have different types (depending on the value you put in). But this can indeed be refactorred 👍